packages/web/src/components/user-badges/UserBadges.tsx

```diff
@@ -105,6 +105,10 @@ const UserBadges = ({
   const audioBadge = tierMap[tier as BadgeTier]
   const hasContent = isVerifiedOverride ?? (isVerified || audioBadge)
 
+  const hasElements = isVerifiedOverride ?? (isVerified || audioBadge)
```

Comment:
these are the same?

Reply:
oh odd it did some weird parethesis, (should be (isVerifiedOverride ?? isVerified) || audioBadge)

Reply:
oh i see what you mean, yup updating